src/OpenLoco/Economy.cpp

```diff
@@ -0,0 +1,15 @@
+#include "Economy.h"
+#include "Interop/Interop.hpp"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::Economy
+{
+    // 0x0046E2C0
+    void sub_46E2C0(uint16_t year)
+    {
+        registers regs;
+        regs.ax = year;
+        call(0x0046E2C0, regs);
+    }
```

Comment:
This will be implemented by #362, but is in need for a do-over. Should be considered out-of-scope.